hibernate-types-52/src/main/java/com/vladmihalcea/hibernate/type/util/JsonbJsonSerializer.java

```diff
@@ -0,0 +1,27 @@
+package com.vladmihalcea.hibernate.type.util;
+
+import org.hibernate.internal.util.SerializationHelper;
+
+import java.io.Serializable;
+
+/**
+ * @author Vlad Mihalcea
+ * @author Jan-Willem Gmelig Meyling
+ */
+public class JsonbJsonSerializer implements JsonSerializer {
```

Comment:
We should have these classes in the jsonb/util folder as they are specific to JSON processing only.